Drop after audio/pulseaudio is updated to 15.0 or later.

meson.build:23:0: ERROR: Dependency lookup for libpulse-mainloop-glib with method 'pkgconfig' failed: Invalid version, need 'libpulse-mainloop-glib' ['>= 16.0'] found '14.2'.

--- meson.build.bak	2023-03-03 21:35:52.000000000 +0000
+++ meson.build	2023-03-05 01:01:04.030577948 +0000
@@ -20,8 +20,8 @@ dependency('libsoup-3.0', version: '>= 3.0')
 dependency('gstreamer-plugins-base-1.0', version: '>= 1.18')
 dependency('gstreamer-play-1.0', version: '>= 1.18')
 dependency('libsoup-3.0', version: '>= 3.0')
-dependency('libpulse-mainloop-glib', version: '>= 16.0')
-dependency('libpulse', version: '>= 16.0')
+dependency('libpulse-mainloop-glib', version: '>= 14.0')
+dependency('libpulse', version: '>= 14.0')
 
 glib_compile_resources = find_program('glib-compile-resources', required: true)
 glib_compile_schemas = find_program('glib-compile-schemas', required: true)
